labman/gui/templates/access.html

```diff
@@ -0,0 +1,91 @@
+{% extends sitebase.html %}
+
+{% block head %}
+<script type='text/javascript'>
+  function removeUser(email, name) {
+    $.post('/auth/access/', {'email': email, 'operation': 'revoke'}, function (data){
+      // Remove contact from additional contact list.
+      $('[id="user-' + email + '"]').remove();
```

Comment:
Minor but, the idiomatic way to search by id is by using `#` i.e. `$('#user-' + email).remove()`. Alternatively is there any reason why you are doing the search like this?